cmd/src/cli/scaffold/rust/lib.rs

```diff
@@ -1,15 +1,75 @@
+#![feature(try_from)]
 #[macro_use]
 extern crate hdk;
 extern crate serde;
+#[macro_use]
 extern crate serde_derive;
 extern crate serde_json;
+#[macro_use]
+extern crate holochain_core_types_derive;
+
+use hdk::{
+    entry_definition::ValidatingEntryType,
+    error::ZomeApiResult,
+};
+use hdk::holochain_core_types::{
+    cas::content::Address, entry::Entry, dna::entry_types::Sharing, error::HolochainError, json::JsonString,
+};
 
 // see https://developer.holochain.org/api/0.0.2/hdk/ for info on using the hdk library
 
+// This is a sample zome that defines an entry type "MyEntry" that can be committed to the
```

Comment:
I don't believe in all the changes from here down, I don't think it'd be better to complete the easy 'entry type' generator thing if we want to make it easier to get boilerplate for this stuff and/or just use the link in the comment to give people a pointer precisely to a place where they can see code very similar to what's here.

Reply:
My goal is having `hc init` and `hc generate` produce something that actually works when you do `hc test`.  I'd rather have this kind of boilerplate with comments and instructions included on how to change it, rather than point people elsewhere, and have them try and figure out where to put all the parts.  Much more error prone and less likelyhood to get started quickly.